src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -22,11 +21,6 @@
     <Compile Include="System\Runtime\Serialization\KnownTypeAttribute.cs" />
     <Compile Include="System.Runtime.Serialization.Primitives.TypeForwards.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'=='netfx'">
-    <Reference Include="mscorlib" />
-    <Reference Include="System.Runtime.Serialization" />
-    <Compile Include="System\Runtime\Serialization\ISerializationSurrogateProvider.cs" />
```

Comment:
Can't we delete this file from our source as well?

Reply:
Looks like we still need this one. In the old project, instead of the source file being included unconditionally, it was being conditionally included for both 'netfx' true and non-'netfx' targets. Now it will become just an unconditional include.